kolibri/plugins/learn/assets/src/views/ContentCard/CardThumbnail.vue

```diff
@@ -4,32 +4,148 @@
     class="card-thumbnail-wrapper"
     :class="{ 'mobile-thumbnail': isMobile }"
     :style="thumbnailBackground"
-  ></div>
+  >
+
+    <ContentIcon
```

Comment:
While I was starting this work, I had refactored the ContentCard component, but I realized it would cause either regressions or breaking changes in other places (namely, Coach). That work got merged as part of string freeze, and now my PR has some of this original code in it that's showing up as an "addition" in the diff. After reviewing, I am reasonably certain that all `ContentCard` references exist in places that have not been refactored as part of hybrid learning - either in the coach plugin, or in components that can probably be removed all together.  I can create a follow up issue for removing the unused components, but it seems like having this still in might be important for not having regression in coach? I'm not familiar enough with the places where swaps would happen to know if that could be done easily as part of this...

Reply:
I don't see anywhere in the coach plugin that is importing any ContentCard components from the learn plugin, so not sure that this caution is needed, unless I'm missing something.

Reply:
You are right, I was not thinking about the plugins :D

Reply:
Actually, I remembered that the current `ChannelCard` builds on the `ContentCard` -- I'll create a follow up issue to pull the dependent styles into the `ChannelCard` and then remove various components and files that aren't being used

Reply:
Follow up https://github.com/learningequality/kolibri/issues/8566

Reply:
Follow up issue here: https://github.com/learningequality/kolibri/issues/8566